src/auth0-sso-login.js

```diff
@@ -1,27 +1,36 @@
-import Auth0Lock from 'auth0-lock';
 import auth0 from 'auth0-js';
-
-// static window interaction, mostly wrapped for enabling easier mocking through unit tests
-export class windowInteraction {
-  static updateWindow(url) {
-    window.location = url;
-  }
-}
+import windowInteraction from './window-interaction';
```

Comment:
`windowInteraction` isn't used in here right?

Reply:
It still is used in `logout` for redirect (`updateWindow`) to auth0 logout url.